app/src/main/java/com/example/musicapp/ui/main/MainViewModel.kt

```diff
@@ -10,8 +10,7 @@ class MainViewModel: ViewModel() {
     var instruments: MutableLiveData<ArrayList<Instrument>> = MutableLiveData<ArrayList<Instrument>>()
     var instrumentService: InstrumentService = InstrumentService()
 
-    fun getAllInstruments(){
+    internal fun getAllInstruments(){
         instruments = instrumentService.fetchInstruments()
-
```

Comment:
I just removed the unnecessary blank line here (Not one of my code review suggestions).